### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by mrm8488**

### CODE REVIEW

File: synthetic-data-save-costs.md [hunk 1]
```diff
@@ -356,9 +356,9 @@ Note that in the [full reproduction script](https://github.com/MoritzLaurer/synt
...
+The main advantage of this data created with the open-source Mixtral model is that the data is fully commercially usable without legal uncertainty. For example, data created with the OpenAI API is subject to the [OpenAI Business Terms](https://openai.com/policies/business-terms), which explicitly prohibits using model outputs for training models that compete with their products and services. The legal value and meaning of these Terms are unclear, but they introduce legal uncertainty for the commercial use of models trained on synthetic data from OpenAI models. Any smaller, efficient model trained on synthetic data could be considered as competing, as it reduces dependency on the API service.
```
Comment (MoritzLaurer) [inline synthetic-data-save-costs.md:359]: I'd keep "prohibit" here, because it refers to the plural "Terms"

**Review by MoritzLaurer — APPROVED** [review]
Thanks @mrm8488 for spotting and correcting the typos! I would just revert back to "prohibit" because it refers to the plural "Terms"

File: synthetic-data-save-costs.md (hunk 1 above)
Comment (pcuenca) [inline synthetic-data-save-costs.md:359]:
```suggestion
The main advantage of this data created with the open-source Mixtral model is that the data is fully commercially usable without legal uncertainty. For example, data created with the OpenAI API is subject to the [OpenAI Business Terms](https://openai.com/policies/business-terms), which explicitly prohibit using model outputs for training models that compete with their products and services. The legal value and meaning of these Terms are unclear, but they introduce legal uncertainty for the commercial use of models trained on synthetic data from OpenAI models. Any smaller, efficient model trained on synthetic data could be considered as competing, as it reduces dependency on the API service. 
```

**Review by pcuenca — APPROVED** [review]
Thanks @mrm8488 !

**Pull request merged**